libr/core/project.c

```diff
@@ -159,6 +159,56 @@ R_API int r_core_project_delete(RCore *core, const char *prjfile) {
 	return 0;
 }
 
+R_API bool r_core_rop_load(RCore *core, const char *prjfile) {
+	char *path, *db;
+	ut8 found = 0;
+	SdbListIter *it;
+	SdbNs *ns;
+
+	Sdb *rop_db = sdb_ns (core->sdb, "rop", false);
+
+	if (!prjfile || !*prjfile) {
+		return false;
+	}
+
+	if (prjfile[0] == '/') {
+		db = r_str_newf ("%s.d", prjfile);
+		if (!db) return false;
+		path = strdup (db);
+	} else {
+		db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
```

Comment:
isnt this path already constructed in other places? like in the r_file apis or so? doesnt sems good to construct the project path by hand all the time.